lib/llm-events/aws-bedrock/bedrock-command.js

```diff
@@ -83,6 +83,11 @@ class BedrockCommand {
       this.isLlama2() === true
     ) {
       result = this.#body.prompt
+    } else if (this.isClaude3() === true) {
+      result = this.#body?.messages?.reduce((acc, curr) => {
+        acc += curr?.content ?? ''
+        return acc
+      }, '')
```

Comment:
We may need to come back and adjust this. It's not clear to me right now how we should handle multiple messages being sent to the LLM.